grid_generation/Gen_CICE_grid.py

```diff
@@ -0,0 +1,96 @@
+"""
+Script: generate_cice_grid.py
+
+Description: 
+This script generates a CICE grid from the MOM super grid provided in the input NetCDF file.
+
+Contact: 
+Name: Ezhilsabareesh Kannadasan
+
+Usage:
+python generate_cice_grid.py <input_superGridPath> <output_file>
+- input_superGridPath: Path to the MOM super grid NetCDF file.
+- output_file: Path to the output CICE grid NetCDF file.
+"""
+import numpy as np
+import xarray as xr
+from netCDF4 import Dataset
+import sys
+
+def generate_cice_grid(in_superGridPath, output_file):
+    # Read input files
+    in_superGridFile = xr.open_dataset(in_superGridPath)
+
+    # Constants
+    ULAT = np.deg2rad(in_superGridFile['y'][2::2, 1::2])
+    ULON = np.deg2rad(in_superGridFile['x'][1::2, 2::2])
```

Comment:
shouldn't this be 

```
ULAT = np.deg2rad(in_superGridFile['y'][2::2, 2::2])
ULON = np.deg2rad(in_superGridFile['x'][2::2, 2::2])
```

?

it might be clearer to write:
`np.deg2rad(in_superGridFile.y.sel(x=slice(start=2,step=2), y=slice(start=2, step=2)))`

?

Reply:
Agreed, but with `isel`, not `sel`